tests/Http/GraphRequestTest.php

```diff
@@ -170,6 +169,11 @@ public function testExecuteAsync()
                          ->executeAsync($this->client);
         $this->assertInstanceOf(GuzzleHttp\Promise\PromiseInterface::class, $promise);
 
+        $promise = $this->requests[0]
+                         ->executeAsync($this->client);
+        $promise2 = $this->requests[2]
+                          ->executeAsync($this->client);
+
```

Comment:
Restored from #437 that removed them without explanation

Reply:
Thanks for catching this. The two extra promises did not seem to be used meaningfully for an assertion. However I notice now that we're trying to unwrap a list of promises & ensure they get fulfilled yielding a GraphResponse obj. Perhaps we could add all the promises to an array so that the loop afterwards is useful?

Reply:
I prefer not touch unrelated code. Even without assertion, this at least "tests" that these extra calls don't throw. Also that re-executes and re-assigns `$promise`, maybe on purpose? So I think better left for another PR :wink:

Reply:
Sure. Not aware of the intention of re-assigning `$promise` in this case but yes this can be handled in another PR